src/controllers/wallet/nano-contracts.controller.js

```diff
@@ -199,11 +199,46 @@ async function getOracleSignedResult(req, res) {
   }
 }
 
+/**
+ * Create an on chain blueprint transaction
+ */
+async function createOnChainBlueprint(req, res) {
+  const validationResult = parametersValidation(req);
+  if (!validationResult.success) {
+    res.status(400).json(validationResult);
+    return;
+  }
+
+  const unlock = lockSendTx(req.walletId);
+  if (unlock === null) {
+    // TODO: return status code 423
+    // we should do this refactor in the future for all APIs
+    res.send({ success: false, error: cantSendTxErrorMessage });
+    return;
```

Comment:
chore(non-blocking): Codecov is informing this error handling flow hasn't been tested.